lib/mini_autobot/google_sheets.rb

```diff
@@ -0,0 +1,77 @@
+module MiniAutobot
+
+  # Class that supports integration with Google Sheets
+  # See documentation here: https://developers.google.com/drive/v3/web/about-auth to set up OAuth 2.0
+  # access to your sheets. Place the info in google_drive_config.json in your config/mini_autobot folder
+  #
+  # To use, use the -g or --google_sheets parameter with the id of your sheet
+  # Example: -g 1gWbpPq7rrTdNtSJSN2ljDlPE-okf0w3w7ai5PlhY8bk
+  class GoogleSheets
+    require 'rubygems'
+    require 'google/api_client'
+    require 'google_drive'
+    require 'json'
+
+    def initialize
+      @session = session
+      @spreadsheet = spreadsheet
+      @worksheet = worksheet
+      @automation_serial_column = automation_serial_column
+      @selected_browser_column = selected_browser_column
+    end
```

Comment:
Great use of initializer methods to move complex logic out of the `#initialize` method. However, if those methods (e.g., `session`, `selected_browser_column`) are meant to be private initializers, consider making them private methods.